### PR TITLE
Fix for #274

### PR DESCRIPTION
**Pull request by pianistrevor**
Add IP Address to serial message in EthernetServerTC constructor

### CODE REVIEW

File: src/Devices/EthernetServer_TC.cpp [hunk 1]
```diff
@@ -27,7 +29,8 @@ EthernetServer_TC* EthernetServer_TC::instance() {
  */
 EthernetServer_TC::EthernetServer_TC(uint16_t port) : EthernetServer(port) {
   begin();
-  serial(F("Ethernet Server is listening on port 80"));
+  IP Address = Ethernet_TC::instance()->getIP();
+  serial(F("Ethernet Server is listening on %i.%i.%i.%i:%i"), IP[0], IP[1], IP[2], IP[3], getPort());
```
Comment (jgfoster) [inline src/Devices/EthernetServer_TC.cpp:33]: Codacy points out that the variable Address is never used. I think `IP` is the "type" and `Address` is the "name".

File: src/Devices/EthernetServer_TC.cpp [hunk 2]
```diff
@@ -3,6 +3,8 @@
 #include <avr/wdt.h>
 
 #include "DateTime_TC.h"
+#include "Devices/Ethernet_TC.h"
+#include "Devices/LiquidCrystal_TC.h"
```
Comment (jgfoster) [inline src/Devices/EthernetServer_TC.cpp:7]: Why was LiquidCrystal_TC.h added?

**Review by jgfoster — COMMENTED** [review]
Correction requested.

File: src/Devices/EthernetServer_TC.cpp (hunk 2 above)
Comment (pianistrevor) [reply src/Devices/EthernetServer_TC.cpp:7]: I closed this yesterday due to these problems. Check the open one.

**Pull request closed without merging**